src/components/Header/index.tsx

```diff
@@ -97,6 +100,25 @@ export const Header: React.FC<HeaderProps> = ({
 
     return items;
   };
+  const getDocsButton = () => {
+    const items = [
+      <div
+        key="documentation-button"
+        aria-label="documentation-button"
+      >
+        <DocumentationButton
+          key="documentation-button"
+          type="link"
+        >
+        </DocumentationButton>
+
+      </div>
+    ];
+    insertPlugins('right', items);
+    if (ClientConfiguration.documentationButtonVisible) {
```

Comment:
I'd move the check to the top of the fiunction:

```
if (!ClientConfiguration.documentationButtonVisible) {
  return;
}
```

Reply:
check [8861c02](https://github.com/terrestris/shogun-gis-client/pull/1743/commits/8861c02552193e756d49bccfefb9548435aadd86)